tests/integration/ambient/baseline_test.go

```diff
@@ -692,6 +720,17 @@ spec:
 				overrideCheck(&opt)
 				src.CallOrFail(t, opt)
 			})
+			t.NewSubTest("port allow").Run(func(t framework.TestContext) {
+				opt = opt.DeepCopy()
+				opt.HTTP.Path = "/allowed-port"
+				opt.Check = check.OK()
+				// sidecar-uncaptured is failing this check
```

Comment:
we already SKipped it above, do we need both?

Reply:
I can't run the tests locally.. I'm hoping my check above will properly skip the sidecar test and then I was going to clean that up